spells.yml

```diff
@@ -3749,7 +3749,7 @@ false_portal:
     The appearance of the window and what lies beyond it are up to you, chosen when you cast the spell. The illusion is only visible from one direction, can only include visual and auditory aspects, can't emit or block light, and can't block noise. For example, you could make a doorway leading to a nonexistent room, a magma-filled crevasse in the floor, or the image of an empty room that you can hide behind. 
 
     A creature can use the Study action to inspect the illusion, making an Investigation check against your spell save DC. On a success, the creature discovers that the illusion is fake.
-  duration: Up to 10 minutes
+  duration: Up to 1 hour
```

Comment:
I'd still like it if were there a built in way to make this last until dispelled, kind of like teleportation circle. Perhaps if it's cast on the same spot every day for a while, it lasts until dispelled, or perhaps if you upcast it at a certain level, it doesn't require concentration after the initial hour and lasts until dispelled?

Reply:
Oh right, I forgot about that. Upcasting might be a good approach, or maybe some sort of consumed material...